scripts/hid_configurator/modules/dfu.py

```diff
@@ -229,6 +229,19 @@ def mcuboot_get_dfu_image_bootloader_var():
     return 'MCUBOOT'
```

Comment:
consider grouping firstly `mcuboot_common_*` functions together and later `mcuboot_*` so they won't intertwine with each other